src/Pulsar.Client/Internal/DeadLetters.fs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Pulsar.Client.Internal
+
+open System
+open Pulsar.Client.Api
+open Pulsar.Client.Common
+open System.Collections.Generic
+open FSharp.Control.Tasks.V2.ContextInsensitive
+open FSharp.UMX
+open System.Threading.Tasks
+
+type internal DeadLettersProcessor
+    (policy: DeadLettersPolicy,
+     getTopicName: unit -> string,
+     getSubscriptionNameName: unit -> string,
+     createProducer: string -> Task<IProducer>) =
+
+    let store = Dictionary<MessageId, ResizeArray<Message>>()
+
+    let producer = lazy (
+
+        let topicName =
+            if String.IsNullOrEmpty(policy.DeadLetterTopic) |> not then
+                policy.DeadLetterTopic
+            else
+                (sprintf "%s-%s-DLQ" (getTopicName()) (getSubscriptionNameName()))
+
+        createProducer topicName
+    )
+
+    let sendAndForgetAsync (builder : MessageBuilder) =
+        task {
+            let! p = producer.Value
+            p.SendAsync(builder) |> ignore
```

Comment:
In Java version, they don't ignore sending error, but returning false on failure, can we stick to that solution?